### ISSUE TITLE
Accessing USB device paths fails in USB relates builds such as ECM--NCM--ACM without apparent reason

### ISSUE THREAD

**Issue by Hammarskjold** [ipxe/ipxe#150]
When using iPXE.EFI and booting from a USB device path the following can be used to access local storage:

file:config.ipxe
file:/EFI/iPXE/config.ipxe
file://DEPLOY/EFI/iPXE/config.ipxe

This all fails in any USB related build when iPXE has initialised the USB devices. Interestingly it works in QEMU, but fails on physical hardware so it seems related to UEFI & USB interaction somehow.

**Comment by Hammarskjold** [comment]
To expand on this. This happens as iPXE takes control of the USB controller, we loose the ability to access the default EFI paths. We have worked around this to use the autoexec.ipxe functionality instead. Considerable work was done to fix above, but as autoexec.ipxe seems to do the trick, the work has not been completed, just 99%.

Closing this until/if the change to auto sanhook the USB drives is iplemented.
